cmd/addressable/addressable.go

```diff
@@ -25,8 +25,8 @@ import (
 func NewCommand() *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "addressable",
-		Short: "Addressable command command",
-		Long:  `Actions related to addressables.`,
+		Short: "Addressable command",
+		Long:  `Actions related to addressable.`,
```

Comment:
I would have recommended using the plural, addressables everywhere ... but that is getting flagged as a spelling error!

Reply:
exactly because of this ... I end up using just "addressable"